src/features/__tests__/Apptest.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { cleanup } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../../app/store';
+import App from '../../App';
+
+afterEach(() => cleanup());
+
+it('App Renders Correctly', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Some of your test are not passing, Kindly make sure all your tests are passing

Reply:
noted, fixed the test cases.